### PR TITLE
DolphinQt/Settings: Replace includes with forward declarations

### PR DESCRIPTION
**Pull request by lioncash**
Avoids dragging in netplay-related headers where they aren't explicitly necessary.

### CODE REVIEW

File: Source/Core/DolphinQt/NetPlay/PadMappingDialog.cpp [hunk 1]
```diff
@@ -3,15 +3,17 @@
...
 
 #include <QComboBox>
 #include <QDialogButtonBox>
 #include <QGridLayout>
 #include <QLabel>
 
+#include "Core/NetPlayClient.h"
+#include "Core/NetPlayServer.h"
+
+#include "DolphinQt/Settings.h"
```
Comment (Techjar) [inline Source/Core/DolphinQt/NetPlay/PadMappingDialog.cpp]: Any reason you moved this include down here?

Comment (lioncash) [reply Source/Core/DolphinQt/NetPlay/PadMappingDialog.cpp]: Yes. It doesn't follow our coding style. Only the header for the source file should be at the top, then it's supposed to follow:

```
<standard includes>

<library includes>

<dolphin includes>
```

Comment (lioncash) [reply Source/Core/DolphinQt/NetPlay/PadMappingDialog.cpp]: i.e. see how MD5Dialog is laid out (that same include is in the correct place).

Comment (Techjar) [reply Source/Core/DolphinQt/NetPlay/PadMappingDialog.cpp]: Ah, I missed the fact that the one above it previous was the header include.

**Pull request merged**